src/InputWithLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const InputWithLabel = ({ id, todoTitle, handleTitleChange, children}) => {
+  const inputRef = React.useRef();
+  React.useEffect(() => {
+    inputRef.current.focus();
+  });
```

Comment:
Good job using the useEffect here to have the input focused always, but is not a good practice having the useEffect without a dependency array, for this requirement is ok, but a better option would be having the code within the useEffect each time the todo list changes (adding a new todo or removing a todo item from the todoList so now  the todoList variable could be a dependency of that side effect, you can pass down the todoList state from the App.jsx file down until you get to the InputWithLabel.jsx file.

Reply:
@gsvidal Thank you!